test.properties

```diff
@@ -7872,7 +7872,7 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
         functional-smoke-bundle-wildfly230-mysql57-jdk21_zulu,\
         functional-smoke-bundle-wildfly261-mysql57-jdk21_zulu,\
         patching-tool-mysql57-jdk8,\
-        playwright-js-bundle-tomcat90-mysql57-jdk8
+        playwright-js-tomcat90-mysql57-jdk21_zulu
```

Comment:
@julschong Why is the `-bundle-` naming convention changed here?

Reply:
@vicnate5 This is the why it was changed https://github.com/liferay-core-infra/liferay-portal/pull/7316#issuecomment-2269922806

did I misunderstand anything?

Reply:
I meant that we can use the non-bundle one to test out the change beforehand.

The final version of the batch for the release tester should be `playwright-js-bundle-tomcat90-mysql57-jdk21_zulu` to follow the naming convention.

Reply:
@vicnate5 Okay, let me change it back